shell/platform/android/io/flutter/plugin/editing/ListenableEditingState.java

```diff
@@ -70,6 +71,14 @@ public Editable getEditable() {
         };
   }
 
+  public ArrayList<TextEditingDelta> getBatchTextEditingDeltas() {
```

Comment:
nit: maybe we should just have a "pop" method that returns the deltas and clear the batched deltas.

Reply:
Yeah that's correct this makes sense then.